include/yaclib/util/result.hpp

```diff
@@ -81,6 +82,8 @@ class Result {
 
   using ValueT = std::conditional_t<std::is_void_v<V>, Unit, V>;
   using VariantT = std::variant<ValueT, std::exception_ptr, E, std::monostate>;
+  template <typename T>
+  using CrefT = std::add_const_t<T>;
```

Comment:
```suggestion
using ConstValue = std::conditional_t<std::is_void_v<V>, void, const V&>;
using MoveValue = std::conditional_t<std::is_void_v<V>, void, V&&>;
```

Reply:
This leads to a compilation error. It's better to write a trait